app/views/bulk_upload_shared/guidance.html.erb

```diff
@@ -0,0 +1,52 @@
+<% content_for :before_content do %>
+  <%= govuk_back_link href: :back %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-l">How to upload logs in bulk</h1>
+
+    <div class="govuk-!-padding-bottom-4">
+      <h2 class="govuk-heading-s">Uploading sales and lettings logs</h2>
+      <p class="govuk-body">You can upload one sales or lettings log at a time, or many at once (known as ‘bulk upload’) with a comma-separated values (CSV) spreadsheet file.</p>
+      <p class="govuk-body">Bulk upload may be easier if your organisation deals with many logs, or if you can export CSV data from your Housing Management System (HMS). If your organisation only deals with a small amount of logs, or you cannot export CSV data, it’s probably easier to enter logs individually.</p>
+      <%= govuk_warning_text text: "You cannot upload lettings and sales logs with the same template - you must export each data type separately, then upload them" %>
+    </div>
+
+    <div class="govuk-!-padding-bottom-4">
+      <h2 class="govuk-heading-s">Creating your CSV files</h2>
+      <p class="govuk-body">To bulk upload successfully, all spreadsheets must be in the correct CSV format.</p>
+      <p class="govuk-body">In most programs, you must resave files as CSV - it’s not usually the default setting. CSV files are also unformatted, so any formatting added before saving (for example colours) will automatically disappear.</p>
+      <%= govuk_details(summary_text: "More about CSV") do %>
+        <p class="govuk-body">A CSV file is a basic spreadsheet with data values in plain text, and columns separated by commas. Each data row is a new text line.</p>
+        <p class="govuk-body">CSV data is easier to process than more common advanced spreadsheet formats, for example Excel. It means CSV is well suited to upload large, or multiple data sets.</p>
+      <% end %>
+    </div>
+
+    <div class="govuk-!-padding-bottom-4">
+      <h2 class="govuk-heading-s">Exporting CSV data</h2>
+      <p class="govuk-body">Export CSV data directly from your current systems, or export then adjust it to CSV.</p>
+      <p class="govuk-body">You can then upload it via a button at the top of the lettings and sales logs pages.</p>
+      <%= govuk_details(summary_text: "My organisation has a CMS") do %>
+        <p class="govuk-body">Some HMS providers sell an add-on "eCORE" module, which exports CSV data for you.</p>
+        <p class="govuk-!-font-weight-bold">It can take HMS providers a while to update these per new collection year, so you may have to wait for updates to export, or adjust your data manually post-export.</p>
+      <% end %>
+      <%= govuk_details(summary_text: "My organisation does not have a CMS") do %>
+        <p class="govuk-body">Your organisation’s IT team may be able to export CSV data for you - find out more about data specifications. This document outlines:</p>
```

Comment:
"Find out more about data specs" should be a link, but that page hasn't been built yet it seems, so this sounds a bit passive aggressive atm. Could link to the spec form as a quick fix, or just wait til the specs are ready. I've added a comment to [this ticket](https://digital.dclg.gov.uk/jira/browse/CLDC-1774) about this.

Reply:
Is this ready for review or should we hold off until you get an answer about that page?

Reply:
This is ready for review now, those spec links are now in this PR too :)